test_util/helpers.py

```diff
@@ -23,81 +24,137 @@ def path_join(p1, p2):
 
 class DcosUser:
     """A lightweight user representation."""
-    def __init__(self, auth_json):
+    def __init__(self, auth_json: dict):
         self.auth_json = auth_json
-        self.auth_header = {}
         self.auth_token = None
         self.auth_cookie = None
+        self.auth = None
 
     def authenticate(self, cluster):
         logging.info('Attempting authentication')
         # explicitly use a session with no user authentication for requesting auth headers
-        if cluster.web_auth_default_user:
-            post = cluster.get_user_session(None).post
-        else:
-            post = cluster.post
-        r = post('/acs/api/v1/auth/login', json=self.auth_json)
+        r = cluster.post('/acs/api/v1/auth/login', json=self.auth_json, auth=None)
         r.raise_for_status()
         logging.info('Received authorization blob: {}'.format(r.json()))
```

Comment:
True. Damn the fools that decided to start using the header called 'Authorization' for authentication everywhere :P